spec/factories/movies.rb

```diff
@@ -14,5 +14,9 @@
     trait :tomorrow do
       start_date { Date.today.advance(days: 1) }
     end
+
+    trait :tendays do
```

Comment:
```suggestion
    trait :eleven_days do
```
I suggest you to change this to make it more semantic